### ISSUE TITLE
Do not refresh the SSL Server Test page while scan is in progress, use AJAX instead

### ISSUE THREAD

**Issue by martinsuchan** [ssllabs/ssllabs-scan#422]
When SSL Server Test is in progress the **web page is periodically calling reload on itself**. This is not right approach because it could lead to various side effects like too many entries in navigation history, losing focus in address bar when trying to navigate to different page and so on.

It would be handy to change the behavior of SSL Server Test to use AJAX or similar technology to **check for the test status without actually reloading the page** and once it's done, just load the result into the DOM of the current page.

**Comment by ivanr** [comment]
Indeed, that would be nice. As it would be fixing the progress estimates :)

**Comment by BenBE** [comment]
Another approach is the one used in https://github.com/BenBE/ssltest/ which uses Server Sent Events (Websockets Light) for live-streaming the new information as they become available. This has the additional advantage of avoiding to continuously open new connections to the server.
